### PR TITLE
Laser and Coil Vektor Rifles

### PR DESCRIPTION
**Pull request by Danice123**

### CODE REVIEW

File: LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini [hunk 1]
```diff
@@ -102,6 +106,10 @@ Bullpup_COIL_SCHEMATIC_SUPPLYCOST=100
 Bullpup_COIL_SCHEMATIC_ALLOYCOST=10
 Bullpup_COIL_SCHEMATIC_ELERIUMCOST=5
 
+Vektor_COIL_SCHEMATIC_SUPPLYCOST=100
+Vektor_COIL_SCHEMATIC_ALLOYCOST;=8
```
Comment (TNychka) [inline LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini]: Typo on the value here

**Review by TNychka — CHANGES_REQUESTED** [review]

**Review by TNychka — DISMISSED** [review]

**Comment by TNychka** [comment]
Can you fix the merge conflicts?

**Comment by Danice123** [comment]
I think I fixed them?

**Comment by TNychka** [comment]
Looks like github was just having issues. Fixed now.

**Pull request merged**